op-chain-ops/genesis/config.go

```diff
@@ -217,13 +215,19 @@ type DeployConfig struct {
 	SuperchainConfigInitiator common.Address `json:"superchainConfigInitiator"`
 	// SuperchainConfigVetoer represents the address of the SuperchainConfigVetoer role.
 	SuperchainConfigVetoer common.Address `json:"superchainConfigVetoer"`
+	// SuperchainConfigGuardian represents the GUARDIAN account in the SuperchainConfig. Has the ability to pause withdrawals.
+	SuperchainConfigGuardian common.Address `json:"superchainConfigGuardian"`
 	// SuperchainConfigDelay represents the time after which an upgrade can occur ifit
 	SuperchainConfigDelay uint64 `json:"superchainConfigDelay"`
 	// SuperchainConfigMaxPause represents the time after which an upgrade can occur ifit
 	SuperchainConfigMaxPause uint64 `json:"superchainConfigMaxPause"`
 	// SequencerKeys represents the set of allowed Sequencer key pairs (batcher hash and unsafe
 	// block signer).
 	SuperchainConfigSequencerKeyPairs []SequencerKeyPair `json:"superchainConfigSequencerKeyPairs"`
+	// Legacy config values.
+	// These are no longer used, but are necessary for parsing legacy deploy-configs
+	PortalGuardian   common.Address `json:"portalGuardian"`
+	FinalSystemOwner common.Address `json:"finalSystemOwner"`
```

Comment:
Thinking about this again, different values are used in the go code vs the solidity code, I am worried that if we leave both it will result in bugs. I would like to have these deduped before going to `develop`

Reply:
I'm not sure I understand, but I've tracked this issue in the PR to `develop`. 

https://github.com/ethereum-optimism/optimism/pull/7656/files#r1358504501